pkg/plugins/backendplugin/manager.go

```diff
@@ -77,13 +82,24 @@ func (m *manager) Register(descriptor PluginDescriptor) error {
 		return errors.New("Backend plugin already registered")
 	}
 
+	pluginSettings := pluginSettings{}
+	if ps, exists := m.pluginSettings[descriptor.pluginID]; exists {
+		pluginSettings = ps
+	}
+
+	hostEnv := []string{
+		fmt.Sprintf("GF_VERSION=%s", setting.BuildVersion),
+		fmt.Sprintf("GF_PLUGIN_ID=%s", descriptor.pluginID),
+	}
+	env := pluginSettings.ToEnv("GF_PLUGIN", hostEnv)
+
```

Comment:
This looks great!

can we add something like:
```
if( license.HasLicense() ) {
  licensePath := fmt.Sprintf("GF_ENTERPRISE_LICENSE_PATH=%s", cfg.EnterpriseLicensePath),
  env = append( env, licensePath )
}
```
Then we have a simple way to support:
https://github.com/grafana/grafana-enterprise/issues/504

Reply:
Added that